eng/helix/content/InstallNode.ps1

```diff
@@ -0,0 +1,60 @@
+
+param(
+    [Parameter(Mandatory = $true)]
+    $Version,
+    
+    [Parameter(Mandatory = $true)]
+    $output_dir
```

Comment:
```suggestion
    $InstallDir
```

Nit: use PowerShell casing conventions. Also, not sure what "output" means in the context of an installer script, so I think "InstallDir" makes a little more sense.